### ISSUE TITLE
ProQuests is disabling after startup

### ISSUE THREAD

**Issue by NeximaOz** [JCulley/ProQuests-Documentation#216]
➤ Spigot account :
➥ https://www.spigotmc.org/members/nexima.40441/

➤ Plugin version :
➥ 2.1.0.2-beta and 2.0.7.1 (I try both versions)

➤ Spigot Version :
➥ 1.12.1

➤ Information :
➥ When server opened firstly proquest work correctly no any error etc. But 2 min later turned to automaticaly red in plugins list. No any error no any message. When try run quest, proquest command etc nothing. Im try proquest different versions but same.
https://prnt.sc/l9ijwz

1. Server Log:
https://hastebin.com/ujusatemey.md

2. I don't know why i'm was getting blocked on github. Maybe i don't know github rules for proquests. But i think i do not wrong thing. I was just want help. Maybe i was open two or three open issue. Sorry for this.

Detailed Analysis:
Im opened server and everything seems good. I'm check logs no any error. Im joined the server and proquests correctly work. Example /quest etc. But 2 or 3 min later not work proquests commands. Just nothing. And im check the "/plugins" list Proquests showing red. Im again check console and logs again no any errors just nothing. Just this. Why ProQuests stopping and not work i no idea.

Note: I was open this ticket but closed. I'm not get help. Also i try latest version but my problem still continue.

**Comment by JCulley** [comment]
1. You are not using the latest version, so support will not be provided until you update.

2. You have many dependency mismatches, which could be the cause of ProQuests disabling.

3. You would need to provide me your log again once these changes have been made provided it still persists.

DO NOT open another issue regarding this, simply reply with your updated information.

**Comment by NeximaOz** [comment]
My ProQuest version latest just i forget version part write.

➤ Plugin version :
➥ 2.1.1

➤ New Server Log :
➥ https://hastebin.com/caqanagifi.md

➤ Picture :
➥ http://prntscr.com/lifysm

Note: Look this picture please. Server new opened. Server when opened firstly ProQuests seem green so working but after 1 min or 2 min disabling i don't know why. We do nothing just im look in console.

**Comment by JCulley** [comment]
> You have many dependency mismatches, which could be the cause of ProQuests disabling.

Please actually read my response entirely prior to responding.

**Comment by NeximaOz** [comment]
You have many dependency mismatches, which could be the cause of ProQuests disabling.

What can i do for this ? Can you explain ?

**Comment by JCulley** [comment]
Use supported dependency versions, from your response I’m led to believe you never ever read your own log file.

**Comment by JCulley** [comment]
Closing as its been over 10 days.

**Comment by NeximaOz** [comment]
My problem still continue. Im check all mismatch versions in log file as you said and im all updated this version mismatch plugins but still same problem. So ProQuest turned to red automatically in /plugins list and not work.

Im getting Version Mismatch warns for this plugins in log;
BATTLE_LEVELS
CITIZENS
HOLOGRAPHIC_DISPLAYS
JOBS
MYTHIC_MOBS
SHOP_GUI_PLUS
VAULT
WORLD_GUARD

But im updated this all plugins. So all latest version.
BattleLevels = Latest (8.2.2)
Citizens = Latest (2.0.24)
Holographic Displays = Latest (2.3.2)
Jobs Reborn = Latest (4.9.5)
MythicMobs = Latest (4.5.0)
ShopGuiPlus = Latest (1.19.1)
Vault = Latest (1.7.1)
WorldGuard = Latest (6.2.2)
and
ProQuest Version = Latest

Log = https://pastebin.com/GSSxtEr7
